scripts/xtensa-build-zephyr.py

```diff
@@ -20,8 +20,8 @@
 
 # Constant value resolves SOF_TOP directory as: "this script directory/.."
 SOF_TOP = pathlib.Path(__file__).parents[1].resolve()
-# Default value may be overriten by -p arg
-west_top = pathlib.Path(SOF_TOP, "zephyrproject")
+west_top = pathlib.Path(SOF_TOP, "..").resolve()
+west_manifest_path = pathlib.Path(SOF_TOP, "west.yml")
```

Comment:
Got it, please add a short comment explaining their purpose: making sure the west workspace is as expected.

Or maybe even better: do these expected values still need to be at the top of the script? It would be much clearer to have them defined only locally where the workspace check is implemented.

Reply:
I removed `west_manifest_path` and made it local, remaining two globals are used everywhere lets keep them for now.